lib/route.coffee

```diff
@@ -207,8 +207,21 @@ class @Route
     # Send response
     endpointContext.response.writeHead statusCode, headers
     endpointContext.response.write body
-    endpointContext.response.end()
-
+    if statusCode in [401, 403]
+      # This is a simple, but not really adequate way to slow down unauthorized scans of the server.
+      # It is not sufficient because hackers can DDOS across multiple threads and systems.
+      # See https://www.owasp.org/index.php/Blocking_Brute_Force_Attacks#Finding_Other_Countermeasures
+      delayInMilliseconds = 500
+      # Hackers can measure the response time to determine things like whether the 401 response was 
+      # caused by bad user id vs bad password.
+      # In doing so, they can first scan for valid user ids regardless of valid passwords.
+      # Delay by a random amount to reduce the ability for a hacker to determine the response time.
+      # See https://en.wikipedia.org/wiki/Timing_attack
```

Comment:
I'm thinking we should only use the second comment. I'm open to debate here, but the first comment seems like it would be better suited if we added in a section to the docs about the current state of security in Restivus, and that could be included in the concerns that haven't been addressed. It doesn't really help to clarify the code any more than the second comment already does. It should also be moved to line 211 where the first comment already is (above this chunk of code).